docs/DeveloperGuide.md

```diff
@@ -479,9 +405,10 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
+   2. Change directory to the folder the jar file is in.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Run the jar file using `java -jar socialbook.jar`.  Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
Should the calendar also be added as part of the GUI?

Reply:
Ur right ill update accordingly